docs/application/web/guides/multimedia/metadata.md

```diff
@@ -0,0 +1,52 @@
+# Metadata
+
+You can extract metadata information from multimedia files.
+
+The main feature of the Metadata API is to extract metadata from a file. You can [extract metadata](#extract-metadata) from multimedia files using the `get()` method.
+
+
+## Prerequisites
+
+To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html), and [tv](../../api/latest/device_api/tv/tizen/metadata.html) applications), the application has to request storage privileges by adding the following privileges to the `config.xml` file:
+
+
+```xml
+<!-- for accessing internal storage only -->
+<tizen:privilege name="http://tizen.org/privilege/mediastorage"/>
+<!-- for accessing external storage only -->
+<tizen:privilege name="http://tizen.org/privilege/externalstorage"/>
+```
+
+
+Additionally, to access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html) and [wearable](../../api/latest/device_api/wearable/tizen/metadata.html) applications), the application has to request [proper storage permissions](../security/privacy-related-permissions.md) using the PPM API.
```

Comment:
Suggestion. How about adding a link to PPM API?

... using the PrivacyPrivilege API (in \[mobile](../../api/latest/device_api/mobile/tizen/ppm.html) and \[wearable](../../api/latest/device_api/wearable/tizen/ppm.html) applications).

If I understood something wrong, please let me know.

Reply:
I agree, it is very good idea.